packages/core/upload/server/controllers/admin-folder-file.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { getService } = require('../utils');
+const { validateDeleteManyFoldersFiles } = require('./validation/admin/folder-file');
+
+module.exports = {
+  async deleteMany(ctx) {
+    const { body } = ctx.request;
+
+    await validateDeleteManyFoldersFiles(body);
+
+    const fileService = getService('file');
+    const folderService = getService('folder');
+
+    const deletedFiles = await fileService.deleteByIds(body.fileIds);
+    const deletedFolders = await folderService.deleteByIds(body.folderIds);
```

Comment:
Should we check if `body.fileIds` and `body.folderIds` exist before calling it? I'm not sure yet what I'll send in the frontend, but in case only files are selected it would be nice, if `folderIds` could be omitted entirely from the request ...